### PR TITLE
Remove IState and update all code and tests

### PR DESCRIPTION
**Pull request by pbirkhol-ni**
- [x] This contribution adheres to [CONTRIBUTING.md](https://github.com/ni/measurement-services-labview/blob/main/CONTRIBUTING.md).

### What does this Pull Request accomplish?

There has been some concern that `IState` adds too much overhead. This change removes `IState` and the corresponding implementations. It then modifies existing VIs to check if the DVR is invalid, which is basically what `IState` was doing anyway.

I only updated one test, which I commented on in the PR. The other test updates were only to the test inputs, where I added "dvr valid" to the json and remove IState.

### What testing has been done?

Automated tests. Some of this code is well automated, others not as much. I also ran an example and verified that it still worked.

### CODE REVIEW

File: Source/Example Measurements/NI-DCPower Source DC Voltage/NIDCPowerSourceDCVoltage.lvproj
Comment (pbirkhol-ni) [inline Source/Example Measurements/NI-DCPower Source DC Voltage/NIDCPowerSourceDCVoltage.lvproj:1]: I need to revert this file.

File: Source/Tests/Tests.Runtime/Measurement Server/Unreserve Sessions/test_AfterCallUnreserveSessions_SessionReservationReturnsErrorForAnyAction.vi
Comment (pbirkhol-ni) [inline Source/Tests/Tests.Runtime/Measurement Server/Unreserve Sessions/test_AfterCallUnreserveSessions_SessionReservationReturnsErrorForAnyAction.vi:1]: I modified this test to also test Register and Unregister, which are new APIs added by Jason.

File: Source/Runtime/Measurement Server.lvproj [hunk 1]
```diff
@@ -129,6 +129,8 @@
 			<Item Name="niFgen_64.dll" Type="Document" URL="niFgen_64.dll">
 				<Property Name="NI.PreserveRelativePath" Type="Bool">true</Property>
 			</Item>
+			<Item Name="Get Connections Dispatcher.vi" Type="VI" URL="../MeasurementLink Session Management/Session Reservation/Get Connections Dispatcher.vi"/>
```
[the remark this answers is not in the document]
Comment (jonathanmendez) [reply Source/Runtime/Measurement Server.lvproj:132]: I've had some luck _sometimes_ with this kind of thing by clearing the compile cache and then reloading the project and/or the relevant VIs.
image

[the remark this answers is not in the document]
Comment (pbirkhol-ni) [reply Source/Runtime/Measurement Server.lvproj:132]: I just cleared the cache, restarted LabVIEW, and those dependencies are still there.

**Review by dixonjoel — APPROVED** [review]

**Pull request merged**